### ISSUE TITLE
Error: Module build failed: BrowserslistError: Unknown browser query `dead`

### ISSUE THREAD

**Issue by A-Sm1th** [parmsang/gatsby-starter-ecommerce#10]
When I launch `gatsby develop`, I get this error:
```uccess delete html and css files from previous builds — 0.021 s
success open and validate gatsby-config.js — 0.012 s
success copy gatsby files — 0.046 s
success onPreBootstrap — 0.343 s
success source and transform nodes — 1.761 s
success building schema — 0.369 s
success createLayouts — 0.009 s
success createPages — 0.050 s
success createPagesStatefully — 0.036 s
success onPreExtractQueries — 0.008 s
success update schema — 0.291 s
success extract queries from components — 0.138 s
success run graphql queries — 0.427 s
success write out page data — 0.004 s
success write out redirect data — 0.001 s
success onPostBootstrap — 0.001 s

info bootstrap finished - 6.197 s

error There was an error compiling the html.js component for the development server.

See our docs page on debugging HTML builds for help https://goo.gl/yL9lND


  Error: Module build failed: BrowserslistError: Unknown browser query `dead` (While processing preset: "/Volumes/HDD/React_Dev_P  rogram/tana_cosmetics/node_modules/babel-preset-env/lib/index.js")

  - index.js:164
    [ecommerce]/[browserslist]/index.js:164:11

  - Array.reduce

  - index.js:132 resolve
    [ecommerce]/[browserslist]/index.js:132:18

  - index.js:224 browserslist
    [ecommerce]/[browserslist]/index.js:224:16

  - targets-parser.js:103 getTargets
    [ecommerce]/[babel-preset-env]/lib/targets-parser.js:103:63

  - index.js:144 buildPreset
    [ecommerce]/[babel-preset-env]/lib/index.js:144:45

  - option-manager.js:317
    [ecommerce]/[babel-core]/lib/transformation/file/options/option-manager.js:317:46

  - Array.map

  - option-manager.js:275 OptionManager.resolvePresets
    [ecommerce]/[babel-core]/lib/transformation/file/options/option-manager.js:275:20

  - option-manager.js:264 OptionManager.mergePresets
    [ecommerce]/[babel-core]/lib/transformation/file/options/option-manager.js:264:10

  - option-manager.js:249 OptionManager.mergeOptions
    [ecommerce]/[babel-core]/lib/transformation/file/options/option-manager.js:249:14

  - option-manager.js:368 OptionManager.init
    [ecommerce]/[babel-core]/lib/transformation/file/options/option-manager.js:368:12

  - index.js:212 File.initOptions
    [ecommerce]/[babel-core]/lib/transformation/file/index.js:212:65

  - index.js:135 new File
    [ecommerce]/[babel-core]/lib/transformation/file/index.js:135:24

  - pipeline.js:46 Pipeline.transform
    [ecommerce]/[babel-core]/lib/transformation/pipeline.js:46:16

  - index.js:46 transpile
    [ecommerce]/[babel-loader]/lib/index.js:46:20

  - fs-cache.js:79
    [ecommerce]/[babel-loader]/lib/fs-cache.js:79:18

  - fs-cache.js:15 ReadFileContext.callback
    [ecommerce]/[babel-loader]/lib/fs-cache.js:15:14
```
Is their something missing in my nodes?

**Comment by parmsang** [comment]
I can't replicate this. Could you try deleting the node_modules directory (and the cache directory if it exists) and reinstall. Also try Yarn and see if it works.

**Comment by A-Sm1th** [comment]
I did delete the `node_modules` and `.cache`, reinstalled them but I still have the same answers.

Regarding `yarn`, I don't know what I should do to try your solution.

**Comment by parmsang** [comment]
The following [issue](https://github.com/gatsbyjs/gatsby/issues/7144) seems related. Could you try using a different version of node? I am using 8.12.0.

**Comment by A-Sm1th** [comment]
Still getting the same error code, even though I have the same Node version than you.

**Comment by isshikivon** [comment]
Same problem here. My code worked fine at first, and I turned off my laptop (before unknowingly updating my Chrome actually, will that matter, seriously?) and the same code suddenly won't work the next day. After deleting `.cache` and `node_modules` and running `npm install` (I tried `yarn install` too, but to no avail), same problem happens again, error messages for my every css file.

**Comment by parmsang** [comment]
Please see https://github.com/gatsbyjs/gatsby/issues/8575 - looks like a fix is expected for gatsby-cli

**Comment by A-Sm1th** [comment]
The fix works with gatsby 2.4.2! 🙌 
`npm i -g gatsby-cli`
